SecretManagement.LastPass.Extension/SecretManagement.LastPass.Extension.psm1

```diff
@@ -61,20 +88,55 @@ function Get-Secret
         $Name = $Matches[1]
     }
 
-    $res = Invoke-lpass 'show','--name', $Name, '--password'
-    if ([string]::IsNullOrWhiteSpace($res)) {
-        $res = Invoke-lpass 'show', '--name', $Name, '--notes'
-    } else {
-        # We have a password, check for a username
-        $username = Invoke-lpass 'show', '--name', $Name, '--username'
-        if ($username) {
-            $res = [System.Management.Automation.PSCredential]::new(
-                $username,
-                (ConvertTo-SecureString $res -AsPlainText -Force))
+    try {
+        $res = Invoke-lpass 'show', '--name', $Name, '--all'
+        if ($null -eq $res) {
+            return 
+        }
+        if ($res[0] -eq 'Multiple matches found.') {
+            Write-Warning "Multiple matches found with the name $Name. `nThe first matching result will be returned."
```

Comment:
I believe Get-Secret can only return one secret. 
I don't have a definitive proof but based on the wildcard discussion on the main project, I assumed the reason they didn't support wildcards was exactly that. 

Get-Secret is meant to return only 1 error.
That being said, I'll validate my assumptions.

Reply:
Yup. First secret only. 

 This cmdlet finds and returns the first secret that matches the provided name. If a vault name is specified, then only that vault will be
    searched. Otherwise, all vaults are searched and the first found result is returned. If a 'Default' vault is specified, then that vault is
    searched before any other registered vault. Secrets that are string or SecureString types are returned as SecureString objects by default.
    Unless the '-AsPlainText' parameter switch is used, in which case the secret is returned as a String type in plain text.

Reply:
@TylerLeonhardt FYI

Get-Secret only return the first secret if multiples are found. 
I attempted to return a list of secrets by changing my `return $output` for a list of 2 secrets and I was able to confirm it first hand.

Reply:
Interesting... I wonder if that's feedback on SecretManagement. In theory a vault could have multiple secrets of the same name...